tequilapi/endpoints/service.go

```diff
@@ -215,17 +259,23 @@ func toServiceRequest(req *http.Request) (*serviceRequest, error) {
 	return &serviceRequest, err
 }
 
-func toServiceInfoResponse(state service.State) serviceInfo {
+func toServiceInfoResponse(instance service.Instance) serviceInfo {
 	return serviceInfo{
-		Status: string(state),
+		Status:   string(service.Running),
+		Proposal: proposalToRes(instance.Proposal()),
+		ID:       string(instance.ID()),
 	}
 }
 
-func toServiceListResponse(state service.State) serviceList {
-	return serviceList{
-		{ID: "6ba7b810-9dad-11d1-80b4-00c04fd430c8", Status: string(state)},
-		{ID: "7037d6f6-1b92-4513-8cb4-8fd1a944b996", Status: string(state)},
+func toServiceListResponse(instances []*service.Instance) (res serviceList) {
+	for _, instance := range instances {
+		res = append(res, serviceInfo{
+			Status:   string(service.Running),
```

Comment:
Starting should happen just before `se.serviceManager.Start`. And I miss `service.Instance` states to be isolated and handled on `service.Manager` side

Reply:
The state is now in the `service.Instance`.